003-other-extensions.sql

```diff
@@ -0,0 +1,11 @@
+create EXTENSION hstore;
+create EXTENSION plperl;
+create EXTENSION hstore_plperl;
+create EXTENSION jsonb_plperl;
+create EXTENSION plperlu;
+create EXTENSION jsonb_plperlu;
```

Comment:
I can not remove plperl since it is required by hstore_plperl

2021-11-08 06:29:59.740 UTC [65] ERROR:  required extension "plperl" is not installed
2021-11-08 06:29:59.740 UTC [65] HINT:  Use CREATE EXTENSION ... CASCADE to install required extensions too.
2021-11-08 06:29:59.740 UTC [65] STATEMENT:  create EXTENSION hstore_plperl;
psql:/docker-entrypoint-initdb.d/003-other-extensions.sql:3: ERROR:  required extension "plperl" is not installed
HINT:  Use CREATE EXTENSION ... CASCADE to install required extensions too.

Reply:
Added your suggested extensions

Reply:
You can remove hstore_plperl too. It's installed on Hyperscale, but there is no way for the user to create it and it is not pre-created.

Reply:
Removed plperl hstore_plperl and jsonb_plperl